CHANGES.md

```diff
@@ -49,6 +49,7 @@ Release Notes.
 * Fix the priority setting doesn't work of the ALS analyzers.
 * Fix bug that `endpoint-name-grouping.yml` is not customizable in Dockerized case.
 * Fix bug that istio version metric type on UI template mismatches the otel rule.
+* Fix `entityId` not taking effect in EqualsAndHashCode annotation
```

Comment:
I think it is `timestamp` is not working.

Reply:
done.